src/modals/SendModals/SendModal.tsx

```diff
@@ -147,24 +152,25 @@ function SendModal<PT extends { fromAddress: Address }, T extends PT>({
         }
       } catch (e) {
         // TODO: When API error codes are available, replace this substring check with a proper error code check
-        const { error } = e as APIError
-        if (error?.detail && (error.detail.includes('consolidating') || error.detail.includes('consolidate'))) {
```

Comment:
Seems like something changed in the API and the errors are not returned in this structure anymore.

Reply:
Has this be discussed with BE team? cc @polarker

Reply:
After a quick sync with BE, it appears that the idea is still to improve the error system and introduce error code to the endpoints. Once that’s done, we will be able to handle error based on error code.

In the meantime, let's try to keep the error messages rather static. @h0ngcha0 do you know if the new structure is used everywhere? Any other error messages have been changed? 

If you plan any other change in the future, thanks in advance for letting us know a little bit in advance so that we can update the way we handle errors 🙏 :)

Reply:
@nop33 Thought: once stabilised, the error mgmt logic could be extracted to the future shared library :)